atomicapp/nulecule/main.py

```diff
@@ -228,6 +228,10 @@ def run(self, cli_provider, answers_output, ask,
         if not self.answers_file:
             self._process_answers()
 
+        # Give more prefernce to the provider that is given via CLI
```

Comment:
grammar :) `Give more preference to the provider when given via CLI`

:smiley: